src/parser/ParserDriver.cpp

```diff
@@ -44,41 +44,23 @@ using YY_BUFFER_STATE = struct yy_buffer_state*;
 extern YY_BUFFER_STATE yy_scan_string(const char*, yyscan_t scanner);
 extern int yylex_destroy(yyscan_t scanner);
 extern int yylex_init_extra(ScannerInfo* data, yyscan_t* scanner);
-extern void yyset_in(FILE* in_str, yyscan_t scanner);
+extern void yyset_lineno(int line_number, yyscan_t scanner);
 extern void yyset_debug(int, yyscan_t scanner);
 
 namespace souffle {
 
-Own<ast::TranslationUnit> ParserDriver::parse(
-        const std::string& filename, FILE* in, ErrorReport& errorReport, DebugReport& debugReport) {
+Own<ast::TranslationUnit> ParserDriver::parse(const std::string& filename, const std::string& code,
+        ErrorReport& errorReport, DebugReport& debugReport) {
```

Comment:
Maybe wait for this fix #2377 to be commited ?

Reply:
Sure, it looks like it would make sense to get that merged that first, and rebase this after.

Reply:
#2377 has been merged, you may update your code accordingly.

Reply:
I updated this MR.